tests/Tgstation.Server.Tests/VersionsTest.cs

```diff
@@ -99,7 +99,14 @@ public void TestDmapiVersion()
 
 			Assert.IsTrue(Version.TryParse(versionLine, out var actual));
 			Assert.AreEqual(expected, actual);
-			Assert.AreEqual(expected, DMApiConstants.Version);
+		}
+
+		public void TestInteropVersion()
```

Comment:
Doesnt this need `[TestMethod]` above it?

Reply:
FUCK